p2p/host.go

```diff
@@ -260,12 +260,14 @@ func NewHost(cfg HostConfig) (Host, error) {
 		libp2p.AutoNATServiceRateLimit(10, 5, time.Second*60),
 	}
 	if cfg.ResourceMgrEnabled {
-		rmgr, err := makeResourceMgr(cfg.ResourceMgrEnabled, cfg.ResourceMgrMemoryLimitBytes, cfg.ResourceMgrFileDescriptorsLimit, cfg.ConnManagerHighWatermark)
+		rmgr, err := makeResourceMgr(false, cfg.ResourceMgrMemoryLimitBytes, cfg.ResourceMgrFileDescriptorsLimit, cfg.ConnManagerHighWatermark)
```

Comment:
```makeResourceMgr(cfg.ResourceMgrDisabled```

Reply:
The `if cfg.ResourceMgrEnabled` condition determines whether resource management is enabled. Also, the first parameter in `makeResourceMgr` now sets infinite limits and it no longer controls enable/disable behavior.